src/d3a/d3a_core/simulation.py

```diff
@@ -71,6 +72,8 @@ def __init__(self):
         self.eta = duration(seconds=0)
         self.elapsed_time = duration(seconds=0)
         self.percentage_completed = 0
+        self.current_slot_str = ""
```

Comment:
This is not used ATM. Were do you plan to use it in the future?

Reply:
I thought it would actually be useful in the future, in order to replace the 'current_market' timestamp that we currently have in the DB. However, you are right, let's introduce this when it is needed. Removing this.